.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
This setup doesn't use anything environment-specific.  It's generalizable so long as we use mocha, so if someone writes TS in vscode, this run configuration can save them the work of figuring out how to debug tests.  I think it adds value, but obviously it's your project so I'll remove it if you still think I should.